arches/app/const.py

```diff
@@ -21,3 +21,4 @@ class ExtensionType(Enum):
     ETL_MODULES = "etl_modules"
     FUNCTIONS = "functions"
     SEARCH_COMPONENTS = "search_components"
+    PERMISSIONS_FRAMEWORKS = "permissions_frameworks"
```

Comment:
As you already discovered, changing the right hand side will cause `module_importer.get_directories` to expect to find permissions modules under my_arches_app/permissions_framework.py instead of permissions.py where you have it now. So either renaming both to PERMISSIONS or adding special case in module_importer I think is needed if we want to change this.

Makes me think we might want a quick test for this after all next to the other ones for that function.

Reply:
I think I'll punt on that change for now in the interest of getting this in.